Meadowlark.net/Meadowlark.Net.Core/Middleware/ParsePathMiddleware.cs

```diff
@@ -0,0 +1,74 @@
+using System.Text.RegularExpressions;
+using Meadowlark.Net.Core.Model;
+
+namespace Meadowlark.Net.Core.Middleware;
+
+public static class ParsePathMiddleware
+{
+  private static string Decapitalize(string str)
```

Comment:
Might want to make this an extension method in the future, if it might be needed elsewhere in the code base.